concepticondata/conceptlists/Castro-2015-608.tsv

```diff
@@ -260,7 +260,7 @@ Castro-2015-608-258	258	home	2406	HOME	家	home, household
 Castro-2015-608-259	259	garden	586	GARDEN	园（菜园）	vegetable plot, garden
 Castro-2015-608-260	260	shadow	1388	SHADE	影子	shadow
 Castro-2015-608-261	261	dream	2374	DREAM	梦	dream
-Castro-2015-608-262	262	ghost	53	SPIRIT	鬼（鬼神）	"ghost, god (a ""large"" spirit - can possess people etc.)"
+Castro-2015-608-262	262	ghost	53	SPIRIT	鬼（鬼神）	ghost, god (a &quot;large&quot; spirit - can possess people etc.)
```

Comment:
Problematic quoting escapes.

Reply:
just " " around large here should be fine.

Reply:
Ah I thought it would be better that way - and there are already other files in concepticon that escape quotes.

Reply:
In the end all of them are converted for the CLLD app anyway, but I think it makes for easier readable lists.

Reply:
Makes sense. It's not in columns that are read very often anyway, but I can change it back. Escaping the quotes only has the advantage that Libre Office doesn't mess it up.